sdk/core/azure-core/src/main/java/com/azure/core/util/paging/ContinuablePagedFlux.java

```diff
@@ -17,6 +20,29 @@
  * @see ContinuablePage
  */
 public abstract class ContinuablePagedFlux<C, T, P extends ContinuablePage<C, T>> extends Flux<T> {
+    final Predicate<C> continuationPredicate;
```

Comment:
Why is this package-private?

Reply:
Was thinking only ContinuablePagedFluxCore would need to access this, at least for now, as it is the only location where paging logic is applied within Core. Currently, as far as I know, there are no sub-types of ContinuablePagedFlux other than ContinuablePagedFluxCore.